app/controllers/api/allocation_events_controller.rb

```diff
@@ -8,8 +8,8 @@ def cancel
       validate_params!(cancel_params)
 
       allocation.transaction do
-        allocation.cancel(**cancellation_details)
-        process_event(allocation.moves, Event::CANCEL, cancel_move_params)
+        allocation.cancel(cancellation_details)
```

Comment:
Not sure about this change - I think it was there to resolve a Ruby deprecation warning. See https://blog.saeloun.com/2019/10/07/ruby-2-7-keyword-arguments-redesign.html#ruby-27

Reply:
oh interesting. okay, will remove